agogosml_cli/cli/templates/{{cookiecutter.PROJECT_NAME_SLUG}}/agogosml-ci-pipeline.yml

```diff
@@ -0,0 +1,126 @@
+resources:
+- repo: self
+
+variables:
+  container_registry: '{{cookiecutter.AZURE_CONTAINER_REGISTRY}}/'
+
+trigger:
+  branches:
+    include:
+    - master
+  paths:
+    include:
+    - '**/agogosml'
+    - '**/input_reader'
+    - '**/output_writer'
+
+jobs:
+- job: Phase_1
+  displayName: Agogosml-Build-CI
+  condition: succeeded()
+  pool:
+    vmImage: 'Ubuntu 16.04'
```

Comment:
do we really need this since the Agent pool is selected on the Build definition as well? this is confusing IMHO and we should direct the user to pick the Hosted Ubuntu - or what if they have their own Ubuntu build agents?

Reply:
Excellent point; dropping this setting.